src/main.ts

```diff
@@ -43,6 +44,12 @@ const init = async (): Promise<void> => {
     const app: Express = express();
     const knexConnection = knex(config.knex);
 
+    const shutDown = async (server: Server): Promise<void> => {
+        await knexConnection.destroy();
```

Comment:
If this was more complex, it would make sense to apply some timeout routine to force exit if it took too long...